features/support/hooks.rb

```diff
@@ -16,6 +18,36 @@
   end
 end
 
+After do |scenario|
+  Sentry.init do |config|
+    config.dsn = 'abc123'
+    config.send_modules = false # list of installed gems
+  end
+
+  if scenario.exception
+    Sentry.with_scope do |scope|
+      scope.set_tags(
+        'cucumber.scenario': scenario.name,
+        'cucumber.failing_step': scenario.test_steps[-1].text
```

Comment:
Note for anyone reading in future: this is incorrect.

I can't find an easy way to identify the step that failed. However, Sentry does a good job of breaking down the backtrace, which has this info in already.